core/src/main/java/com/gentics/mesh/core/endpoint/schema/SchemaCrudHandler.java

```diff
@@ -329,4 +339,34 @@ public void handleApplySchemaChanges(InternalActionContext ac, String schemaUuid
 
 	}
 
+	/**
+	 * Handle the request for the project, linked to the schema with the given UUID.
+	 * 
+	 * @param ac
+	 * @param schemaUuid
+	 */
+	public void handleGetLinkedProjects(InternalActionContext ac, String schemaUuid) {
+		validateParameter(schemaUuid, "schemaUuid");
+
+		PagingParameters pagingInfo = ac.getPagingParameters();
+		ValidationUtil.validate(pagingInfo);
+
+		utils.syncTx(ac, tx -> {
+			SchemaDao schemaDao = tx.schemaDao();
+			HibSchema schema = schemaDao.loadObjectByUuid(ac, schemaUuid, READ_PERM);
+
+			Result<HibProject> result = schemaDao.findLinkedProjects(schema);
```

Comment:
This will not handle permissions on the projects (so it will return all projects the schema is assigned to, regardless of the permissions)